main.go

```diff
@@ -338,6 +341,25 @@ func main() {
 	<-cleanupDoneCh
 }
 
+func handleLogChangeSignal(ctx context.Context, ch chan os.Signal, logLevel logrus.Level) {
+OUT:
+	for {
+		select {
+		case <-ctx.Done():
+			break OUT
+		case sig := <-ch:
+			switch sig {
+			case syscall.SIGUSR1:
+				log.FromContext(ctx).Infof("Setting log level to %s", logrus.TraceLevel.String())
+				logrus.SetLevel(logrus.TraceLevel)
+			case syscall.SIGUSR2:
+				log.FromContext(ctx).Infof("Setting log level to %s", logLevel.String())
+				logrus.SetLevel(logLevel)
+			}
+		}
+	}
+}
```

Comment:
Also, I wonder, does it make sense to run the pprof server in case of enabling trace level via signals?

cc @szvincze

Reply:
@denis-tingaikin: Since the TRACE log level could have influence on the different profiles, like CPU and memory, I would not tie the log level and profiling together but keep them separately configurable. If someone needs both activated then configure them one by one.